doc/modules/model_evaluation.rst

```diff
@@ -288,6 +288,166 @@ parameter:
+
+.. :math::
+
+    \bar{S} = \frac{1}{n_\text{samples}}
+              \sum_{i=0}^{n_\text{samples}-1} S(y_i, \hat{y}_i).
+
+The optimal point prediction :math:`\hat{y}^\star` under :math:`S` is the
+*Bayes Rule*
+
+.. :math:
+
+    \hat{y}^\star = \operatorname{argmax}_x \mathbb{E}[S(Y, x)].
+
+Note that in order to get an unbiased estimate of :math:`\mathbb{E}[S(Y, x)]`
+for model evaluation and model selection, one has to use a test set independent
+of the training set.
+
+If you are free to choose a scoring function, you should first ask yourself,
+which functional of :math:`F(Y)` you want to predict as a point forecast:
+the mean, the mode, the median, a quantile, ...
+Then, **strictly consistent** scoring functions for this functional are to be
+preferred because only they provide that :math:`\hat{y}^\star` is unique and
+equals the desired target functional, see [Gneiting2009]_ for more details.
+Stated otherwise: if the scoring function is (strictly) consistent for the
+functional at interest, this functional is the (unique) optimal point forecast
+under this scoring function.
+
+One could say that consistent scoring functions act as *truth serum* in that
+they guarantee "that truth telling [. . .] is an optimal strategy in
+expectation" [Gneiting2014]_.
+
+Here, we list some of the well known functionals and corresponding scoring
+functions. For further criteria on how to select a specific one, see
+[Fissler2022]_.
+
+==================    ========================    ===================   ================   ==============================
+functional            scoring or loss function    property              response y         prediction
+==================    ========================    ===================   ================   ==============================
+**Classification**
+mean                  Brier score                 strictly consistent   multi-class        ``predict_proba``
+mean                  log loss                    strictly consistent   multi-class        ``predict_proba``
+median                absolute error              strictly consistent   binary             ``predict_proba``
+mode                  zero-one loss               strictly consistent   multi-class        ``predict``
+**Regression**
+mean                  squared error               strictly consistent   all reals          ``predict``, all reals
+mean                  Poisson deviance            strictly consistent   non-negative       ``predict``, stritcly positive
+mean                  Gamma deviance              strictly consistent   stricly positive   ``predict``, stritcly positive
+median                absolute error              strictly consistent   all reals          ``predict``, all reals
+quantile              pinball loss                strictly consistent   all reals          ``predict``, all reals
+==================    ========================    ===================   ================   ==============================
```

Comment:
The column "property" in this table seems unnecessary, for it has the same value for all the entries. What about introducing the table with: "Here we list some of the well known functionals and corresponding consistent scoring functions.", and removing the "property" column?
Also, it would be useful to link the scoring or loss function to the related scikit-learn documentation when possible.